superset/security/manager.py

```diff
@@ -488,7 +499,7 @@ def get_schemas_accessible_by_user(
 
         return [s for s in schemas if s in accessible_schemas]
 
-    def get_datasources_accessible_by_user(
+    def datasources_accessible_by_user(
```

Comment:
Per https://github.com/apache/incubator-superset/pull/10030 I think this should remain `get_datasources_accessible_by_user`.